### PR TITLE
propose to refine Charter Update process

### PR DESCRIPTION
**Pull request by xueyuanjia**
specify what/what/when to update group charters following the discussions [[1](https://github.com/w3c/charter-drafts/issues/259), [2](https://github.com/w3c/charter-drafts/pull/272#issuecomment-644377948)].

### CODE REVIEW

File: process/charter.html [hunk 1]
```diff
@@ -372,6 +370,13 @@ <h4>Updating the Charter End Date in the Charter</h4>
 (e.g., on the group's site) to say something like
 "Note: See changes from this initial schedule on the group home page."</li>
 </ul>
+	<li>When a group <strong>Chair</strong> (re)appointed or resigned, shortly before/after the announcements to w3c-ac-members@w3.org and cc's chairs@w3.org (sample emails [<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=appointed&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">1</a>][<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=steps+down&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">2</a>]), Team Contact (or Comm Team) modifies the charter, including:
```
Comment (wseltzer) [inline process/charter.html:373]: s/(re)appointed or resigned/is (re)appointed or resigns/

**Review by koalie — APPROVED** [review]
+1 with Wendy's proposed fix. Thanks! (and sorry for missing this review request)

**Comment by xueyuanjia** [comment]
Hi @plehegar, does the proposed update look good to you? If approved, I will notify w3t about the changes.

File: process/charter.html (hunk 1 above)
Comment (plehegar) [reply process/charter.html:373]:
```suggestion
	<li>When a group <strong>Chair</strong> (re)appointed or resigns, shortly before/after the announcements to w3c-ac-members@w3.org and cc's chairs@w3.org (sample emails [<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=appointed&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">1</a>][<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=steps+down&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">2</a>]), Team Contact (or Comm Team) modifies the charter, including:
```

Comment (plehegar) [inline process/charter.html:373]:
```suggestion
	<li>When a group <strong>Chair</strong>  is (re)appointed or resigned, shortly before/after the announcements to w3c-ac-members@w3.org and cc's chairs@w3.org (sample emails [<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=appointed&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">1</a>][<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=steps+down&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">2</a>]), Team Contact (or Comm Team) modifies the charter, including:
```

Comment (plehegar) [reply process/charter.html:373]:
```suggestion
	<li>When a group <strong>Chair</strong> is (re)appointed or resigns, shortly before/after the announcements to w3c-ac-members@w3.org and cc's chairs@w3.org (sample emails [<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=appointed&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">1</a>][<a href="https://www.w3.org/Search/Mail/Member/search?keywords=&hdr-1-name=subject&hdr-1-query=steps+down&index-grp=Member__FULL+Public__FULL&index-type=t&type-index=w3c-ac-members">2</a>]), Team Contact (or Comm Team) modifies the charter, including:
```

**Review by plehegar — CHANGES_REQUESTED** [review]

**Pull request merged**